packages/frontend/src/pages/SavedDashboards.tsx

```diff
@@ -94,33 +101,41 @@ const SavedDashboards = () => {
                         />
                     </PageBreadcrumbsWrapper>
 
-                    {userCanManageDashboards && !isDemo && (
-                        <Button
-                            text="Create dashboard"
-                            icon="plus"
-                            loading={isCreatingDashboard}
-                            onClick={() =>
-                                createDashboard({
-                                    name: DEFAULT_DASHBOARD_NAME,
-                                    tiles: [],
-                                })
-                            }
-                            disabled={hasNoSpaces}
-                            title={
-                                hasNoSpaces
-                                    ? 'First you must create a space for this dashboard'
-                                    : ''
-                            }
-                            intent="primary"
-                        />
-                    )}
+                    {userCanManageDashboards &&
+                        !isDemo &&
+                        (dashboards.length > 0 || hasNoSpaces) && (
```

Comment:
Not sure why we need this part of the condition. What scenarios is it filtering out the create button ?